### PR TITLE
[Enhancement] support check page statistics from profile

### PR DESCRIPTION
**Pull request by zombee0**
Fixes #issue
截屏2023-08-02 21 02 45
截屏2023-08-02 21 03 02

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4

### CODE REVIEW

File: be/src/formats/parquet/column_chunk_reader.cpp [hunk 1]
```diff
@@ -105,6 +105,9 @@ Status ColumnChunkReader::_parse_page_header() {
     if (_page_reader->current_header()->type == tparquet::PageType::DATA_PAGE) {
         const auto& header = *_page_reader->current_header();
         _num_values = header.data_page_header.num_values;
+        _opts.stats->has_page_statistics |=
+                (header.data_page_header.__isset.statistics && (header.data_page_header.statistics.__isset.min_value ||
+                                                                header.data_page_header.statistics.__isset.min));
```
Comment (DorianZheng) [inline be/src/formats/parquet/column_chunk_reader.cpp:110]: do we need to check if max exist or not

Comment (zombee0) [reply be/src/formats/parquet/column_chunk_reader.cpp:110]: I think it's ok, usually set min/max together, just check min or max is enough.

**Review by dirtysalt — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Pull request merged**